src/test/java/Pages/LeavePage.java

```diff
@@ -0,0 +1,50 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LeavePage {
```

Comment:
In this page coding standers are not followed Revised you whole class before pushing it again

Reply:
Revised Standers in a proper way